ir/globals.h

```diff
@@ -72,7 +72,14 @@ extern bool has_null_block;
 /// Whether the programs do memory accesses that load/store int/ptrs
 extern bool does_int_mem_access;
 extern bool does_ptr_mem_access;
-extern bool does_ptr_store;
+
+struct PtrStoreInfo {
```

Comment:
why this complication (vs adding another bool)?
Is there other info you expect to collect from stored ptrs?

Reply:
Yes. I'm going to collect whether there is any local pointer that is stored, so a struct was used here.

Reply:
I'm not strongly in favor of having structs here, so fixed it to be const bools instead